forge-gui/res/cardsfolder/upcoming/final_flourish.txt

```diff
@@ -0,0 +1,8 @@
+Name:Final Flourish
+ManaCost:1 B
+Types:Instant
+K:Kicker:Sac<1/Creature;Artifact/creature or artifact>
+A:SP$ Pump | ValidTgts$ Creature | TgtPrompt$ Select target creature | NumAtt$ -X | NumDef$ -X | IsCurse$ True | SpellDescription$ Target creature gets -2/-2 until end of turn. If this spell was kicked, that creature gets -6/-6 until end of turn instead.
```

Comment:
Probably `StackDescription$ REP Target creature_{c:Targeted}` will be easiest way to get good StackDesc